gen/dcos-config.yaml

```diff
@@ -678,7 +678,7 @@ package:
       {
         "role": "master",
         "pull": true,
-        "diagnostics-bundle-dir": "/var/lib/dcos/dcos-diagnostics/diag-bundles",
+        "diagnostics-bundle-dir": {{ dcos_diagnostics_bundles_dir }},
```

Comment:
This needs to be in quotes so that it renders as a valid JSON string:
```
        "diagnostics-bundle-dir": "{{ dcos_diagnostics_bundles_dir }}",
```